sdk/postgresql/arm-postgresql-flexible/README.md

```diff
@@ -1,11 +1,11 @@
-## Azure PostgreSQLFlexibleManagementClient SDK for JavaScript
+## Azure PostgreSQLManagementClient SDK for JavaScript
```

Comment:
@deyaaeldeen @sarangan12 Do you know why the client name is changed here?
The original one is generated from https://github.com/Azure/azure-rest-api-specs/tree/master/specification/postgresql/resource-manager#tag-package-2021-04-10-privatepreview
The current one is generated from https://github.com/Azure/azure-rest-api-specs/tree/master/specification/postgresql/resource-manager#tag-package-2021-03-31-privatepreview